src/Contrib.KafkaFlow.ProcessManagers.SqlServer/schema/0001.Schema.sql

```diff
@@ -0,0 +1,5 @@
+IF NOT EXISTS(SELECT * FROM sys.databases WHERE name = 'kafkaflowtest')
+BEGIN
+	CREATE DATABASE [kafkaflowtest]
```

Comment:
I think this should be setting the schema and not the DB name. As per my comment for the `outbox` table